### ISSUE TITLE
Q3 Jacobian and non-trivial stability

### ISSUE THREAD

**Issue by rredelmeier** [bbolker/math3mb#24]
I think there might be an issue with the Jacobian, and therefore the non-trivial stability analysis, for question 3? I think the bottom right term of the Jacobian should be $\frac{q*C*V}{r}$, and if you plug in the non-trivial equilibrium, you wind up T=$ 1+ \frac{ln(r)}{r(1-\frac{1}{r})}$. This is less than one nowhere, since the second term is positive for any r>0.

That seems weird, since it implies the non-trivial equilibrium is never stable -- is it possible for you to take a second look with a corrected Jacobian?

**Comment by bbolker** [comment]
You're correct about the bottom-right term (will correct shortly), and also correct that the non-trivial equilibrium is never stable.  This is one of the interesting/surprising things about the Nicholson-Bailey model.
